kcidb/orm/__init__.py

```diff
@@ -3,1287 +3,27 @@
 objects, but without the object-oriented interface.
 """
 
-import re
-import textwrap
 import logging
-import argparse
 from abc import ABC, abstractmethod
-import jsonschema
 import kcidb.io as io
 import kcidb.misc
 from kcidb.misc import LIGHT_ASSERTS
+from kcidb.orm.query import Pattern, PatternHelpAction
+from kcidb.orm.data import SCHEMA
```

Comment:
Could you please import the `query` and `data` modules here, instead of the symbols directly? And refer to the symbols through them, e.g. `query.Pattern` and `data.SCHEMA`. First, this would satisfy #404, and then it would stop these symbols appearing like they're actually still defined here. It's best to always refer to the symbols by their canonical name and definition location, and not use third-party references (e.g. like the `Pattern` imported here would be).

In other words, the symbols you **moved** out of the `kcidb.orm` module should no longer be available there.

This will also require updating more code in other modules, including `main.py` in the top directory, containing the Cloud Function entry points.

Reply:
Okay